src/screens/Analytics/GlobalSearch/GlobalSearchBar.res

```diff
@@ -158,6 +158,7 @@ let make = () => {
         (ctrlKey && keyPressed == global_search_activate_key)
     ) {
       setShowModal(_ => true)
+      event->preventDefault
```

Comment:
is it required last time this was causing issue right?

Reply:
last time it was called for everykey press caused the issue, now its inside the condition statement

Reply:
if it is cmd+k then only called or else not